mantidproject/mantid
# IN16B instrument configuration

**Description of work.**
This PR implements for IN16B:

- secondary PSD configuration, with the first tube angle at 33.1 degree (plus its test)
- fixes the analyser distance for the single detectors through IDF (system test reference updated)
- flips the entire detector to the correct side wrt the beam (this has no real effect)
- removes .map grouping file, changes .xml grouping file to use `detids` instead of `ids`
- adds more rules in run merging behaviour through parameters file

**To test:**

1. Load the run 247933.nxs in the unit tests data, under ILL/IN16B.
2. Right click > show detectors.
3. Scroll down to the pixel with ID=65, two theta should be 33.1 degrees.
4. Scroll down further, the pixels with ID=2049 and 2050 should have R = 4 meters.



*There is no associated issue.*

*Release notes have been updated.*

The IDF change is done with the updated generator script, so

**Merge also PR [#140](https://github.com/mantidproject/mantidgeometry/pull/140) in mantidgeometry.**

---

#### Reviewer ####



Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
